tests/ember-electron/main.js

```diff
@@ -47,15 +47,3 @@ app.on('ready', function onReady() {
     mainWindow = null;
   });
 });
-
-let cwd = __dirname || resolve(dirname());
```

Comment:
I rm’ed it because I ran ember g ember-electron and it was rm’ed. IIRC this behaviour is tested in the unit tests; if it isn’t it ought to live there vs. generated code.